### PR TITLE
 turning of mt filtering for sv

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- Turning off filtering for MT SV variants 

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by henrikstranneheim — APPROVED** [review]
Well done! Was not easy following the filtering and pipe splitting. Feels important with some testing and screenshots to make sure all goes as intended.
There are some github annotations as well to fix

File: templates/grch38_mip_rd_dna_config.yaml [hunk 1]
```diff
@@ -66,6 +66,11 @@ vcfanno_config: grch38_vcfanno_config_-v0.2-.toml
 ### Analysis
 ## Programs
 ## Parameters
+sv_fqa_vcfanno_filters:
+  - GNOMADAF
+  - GNOMADAF_POPMAX
```
[the remark this answers is not in the document]
Comment (jemten) [reply templates/grch38_mip_rd_dna_config.yaml:71]: You're right we annotate with the key GNOMADAF_popmax.

**Comment by jemten** [comment]
@henrikstranneheim,  I wrote a small test to check if all the requested vcfanno frequency filters were in the toml file. This only warns the user as it feels like a little too late to fail and exit once jobs have been submitted. Ideally this check should be done with all the other pipeline checks.  You think it's worth the effort to move the check?

**Comment by jemten** [comment]
MIP config
```yaml
  sv_svdb_query_db_files:
    # FORMAT; filename|OUT_FREQUENCY_INFO_KEY|OUT_ALLELE_COUNT_INFO_KEY|IN_FREQUENCY_INFO_KEY|IN_ALLELE_COUNT_INFO_KEY|USE_IN_FREQUENCY_FILTER
    grch37_gnomad.genomes_-r2.1.1_sv-.vcf.gz: gnomad_sv|AF|AC|AF|AC|1
    grch37_loqusdb_sv_variants_export-20210416-.vcf: clinical_genomics_loqus|Frq|Obs|Frq|Obs|1
    grch37_mip_sv_svdb_export_-2018-10-09-.vcf: clinical_genomics_mip|AF|OCC|FRQ|OCC|1
    grch37_svdb_query_decipher_-v1.0.0-.vcf: decipher|AF|OCC|FRQ|OCC
    grch37_svdb_query_clingen_cgh_benign_-v1.0.0-.vcf: clingen_cgh_benign
    grch37_svdb_query_clingen_cgh_pathogenic_-v1.0.0-.vcf: clingen_cgh_pathogenic
    grch37_svdb_query_clingen_ngi_-v1.0.0-.vcf: clingen_ngi|AF|OCC|FRQ|OCC|1
    grch37_swegen_concat_sort_-20170830-.vcf: swegen|AF|OCC|FRQ|OCC|1
```

MIP parameter file with default
```
sv_fqa_vcfanno_filters:
  associated_recipe:
    - sv_annotate
  data_type: ARRAY
  default:
    - GNOMADAF
    - GNOMADAF_popmax
  type: recipe_argument
```

Annotation toml file 
```toml
#TOML

title = "Vcfanno configuration file"

[[annotation]]
file="/path/grch37_gnomad_reformated_-r2.1.1-.vcf.gz"
fields = ["AF", "AF_popmax"]
ops=["self", "self"]
names=["GNOMADAF", "GNOMADAF_popmax"]

[[annotation]]
file="/path/grch37_gnomad_genomes_mt_-r3.1-.vcf.gz"
fields = ["AF_hom", "AF_het", "mitotip_score", "mitotip_trna_prediction"]
ops=["self", "self", "self", "self"]
names=["MTAF_hom", "MTAF_het", "mitotip_score", "mitotip_trna_prediction"]
```

Exclude filters on bcftools filter 
```
--exclude "INFO/GNOMADAF > 0.40 | INFO/GNOMADAF_popmax > 0.40 | INFO/clingen_ngiAF > 0.40 | INFO/clinical_genomics_mipAF > 0.40 | INFO/swegenAF > 0.40 | INFO/gnomad_svAF > 0.40 | INFO/clinical_genomics_loqusFrq > 0.40 "
```

**Pull request merged**